apps/institution-service/.env.example

```diff
@@ -0,0 +1,10 @@
+AUTH0_AUDIENCE=
+AUTH0_DOMAIN=
+
+# Cypress testing variables
+E2E_CLIENT_ID=
```

Comment:
We should keep cypress out of our main env files. 

Example: https://github.com/Universal-Connect-Project/ucp-hosted-app/blob/main/apps/ucp-ui/cypress.example.env.json

Reply:
I saw that and thought it was kinda confusing because you look at the [cypress.config.ts](https://github.com/Universal-Connect-Project/ucp-hosted-app/blob/89cabb8f1799da3fca0f2bd67326b5edb86c96df/apps/institution-service/cypress.config.ts#L6-L11) and it looks like all the env variables are coming from the .env file. And then locally you have a cypress.example.env which overwrites those variables. But on the github that file doesn't exist so it just pulls the variables from the environment variables.